collect_app/src/androidTest/java/org/odk/collect/android/support/pages/NotificationDrawer.kt

```diff
@@ -100,16 +111,17 @@ class NotificationDrawer {
         }
 
         device.wait(Until.gone(By.text("No notifications")), 1000L)
-        closeNotificationDrawerIfOpened()
-        isOpen = false
+        close()
```

Comment:
Why here it is not `ensureClosed`? Doesn't the notification drawer close after clicking on `Clear all`?